service/src/main/java/uk/gov/hmcts/reform/fpl/utils/EmailNotificationHelper.java

```diff
@@ -27,6 +35,20 @@ public static String buildSubjectLine(final CaseData caseData) {
             .collect(joining(", "));
     }
 
+    public static String buildSubjectLineWithHearingBookingDateSuffix(final String subjectLine,
+                                                                      final List<Element<HearingBooking>>
+                                                                          hearingDetails) {
+        String hearingDate = "";
+        if (isNotEmpty(hearingDetails)) {
+            hearingDate = " hearing " + dateFormatterService.formatLocalDateToString(
```

Comment:
I feel like a nice private helper method would read better here i.e hearingDate = `" hearing " + getFormattedNextHearingDate();`

Reply:
good suggestion. But was trying to make as minimum change here as possible. This is because method was moved to _EmailNotificationHelper_ for reusability.